scylla/datadog_checks/scylla/data/conf.yaml.example

```diff
@@ -123,3 +123,26 @@ instances:
     #
     # extra_headers:
     #   <HEADER_NAME>: <HEADER_VALUE>
+
+## Log Section (Available for Agent >=6.0)
+##
+## type - mandatory - Type of log input source (tcp / udp / file / windows_event)
+## port / path / channel_path - mandatory - Set port if type is tcp or udp. Set path if type is file. Set channel_path if type is windows_event
+## service - mandatory - Name of the service that generated the log
+## source  - mandatory - Attribute that defines which Integration sent the logs
+## sourcecategory - optional - Multiple value attribute. Used to refine the source attribute
+## tags: - optional - Add tags to the collected logs
+##
+## Discover Datadog log collection: https://docs.datadoghq.com/logs/log_collection/
+#
+# logs:
+#   - type: file
+#     path: <LOG_FILE_PATH>
+#     source: scylla
+#     sourcecategory: database
+#     service: <SERVICE>
+#     #To handle multi line that starts with yyyy-mm-dd or yyyy-dd-mm use the following pattern
+#     log_processing_rules:
```

Comment:
Does Scylla always requires multi line processing? The readme keeps it commented out 🤔 

Can we have consistency between the Readme and the conf.yaml?